services/QuillLMS/app/services/demo/report_demo_creator.rb

```diff
@@ -403,15 +402,14 @@ def self.create_activity_sessions(students, classroom)
           cu = ClassroomUnit.find_by(classroom_id: classroom.id, unit_id: unit.id)
           act_session = ActivitySession.create({activity_id: act_id, classroom_unit_id: cu.id, user_id: student.id, state: "finished", percentage: temp.percentage})
 
-          temp.old_concept_results.each do |cr|
+          temp.concept_results.each do |cr|
             values = {
               activity_session_id: act_session.id,
               concept_id: cr.concept_id,
-              metadata: cr.metadata,
-              question_type: cr.question_type
+              metadata: cr.legacy_format[:metadata],
+              question_type: cr.concept_result_question_type&.text
```

Comment:
Is this expected that question_type uses &.text whereas the report_demo_ap_creator and create_admin_report do not.